lib/middleware/user-context.js

```diff
@@ -21,10 +22,12 @@ module.exports = function userContextMiddleware() {
     loopbackContext.set('accessToken', req.accessToken.id);
     const app = req.app;
     const UserModel = app.accessUtils.options.userModel || 'User';
+    const GroupAccessModel = app.accessUtils.options.groupAccessModel || 'Team';
```

Comment:
Almost/ I don't think it should default to 'Team' either. If you make it the following I'll merge:

```
const GroupAccessModel = app.accessUtils.options.groupAccessModel || 'GroupAccess';
```

Thanks!